Net24Solution/Net24Task/GoogleConfig.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Net24Task
+{
+    public class GoogleConfig : WebSiteData
```

Comment:
classes {vendor}Config have no difference between each other. They are not needed. 
Keep only WebSiteConfig class. 
In order to load multiple websites, use an array like you have at appsettings2.json.